src/styles/SearchForm.css

```diff
@@ -0,0 +1,50 @@
+.search--form input[type="text"] {
```

Comment:
You could just put a className on this element instead of using a highly specific selector like this, generally you dont want to use a more specific selector than is necessary